module-1/pipelines-project/your-code/CODE.py

```diff
@@ -1,6 +1,7 @@
 import pandas as pd
```

Comment:
1. Well done to use relative paths instead of absolute paths. Thanks for that. 
2. You may include the title of the plot in your reporting function instead of including it on your main. Come to me if you want to know how. You would need to do some research on maptplotlib or seaborn for this. 
3. The exercises asked for the following: 
"You must demonstrate all the topics we covered in the chapter (functions, list comprehensions, string operations, and error handling) in your processing of the data." 
You haven't included any of them with the exception of functions.